54.Transformations/main.cpp

```diff
@@ -648,7 +676,7 @@ class TransformationApp : public ApplicationBase
 			timestamp++;
 
 			// safe to proceed
-			cb->begin(IGPUCommandBuffer::EU_NONE);
+			cb->begin(video::IGPUCommandBuffer::EU_NONE);
```

Comment:
while the example compiles, an assertion triggers  
```
cmdbuf->getRecordingFlags().hasFlags(IGPUCommandBuffer::EU_ONE_TIME_SUBMIT_BIT)
```

Reply:
yeah the commandbuffer used for asset converter MUST now be created as a ONE_TIME_SUBMIT type